test/simulator/misc/choice-parser.js

```diff
@@ -155,6 +140,27 @@ describe('Choice parser', function () {
 			});
 		});
 
+		it('should allow mega evolving and targeting in the same move in either order', function () {
```

Comment:
Hm, you have placed it outside the inner ``describe`` blocks :|

Reply:
Oh man, I'm doing really bad at placing this in the correct place.

Reply:
Also, this could well be tested in a Singles game. I am not sure why you create a Doubles one...

Reply:
Hopefully the latest commit finally found the correct spot for this.